src/core/server/stacks/approveComment.ts

```diff
@@ -71,37 +69,4 @@ const approveComment = async (
   return result.after;
 };
 
-const enableRepliesToChildren = async (
-  commentID: string,
-  mongo: MongoContext
-) => {
-  const children = await mongo
-    .comments()
-    .find({
-      parentID: commentID,
-    })
```

Comment:
This whole solution of tagging with `rejectedAncestor` would have worked pretty well if this filter was more specific. Namely, if it had `tenantID`, `storyID`, and `parentID` we likely wouldn't have had a performance concern.

I could have just added these filters, however, I opted to do the pure read `ancestorID` + rejected state lookup I did earlier because it avoids doing any writing of state to comments to get the same result.